src/Symfony/Component/Debug/CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ CHANGELOG
 5.0.0
 -----
 
-* removed `FlattenException::setTraceFromException()` in favor of `setTraceFromThrowable()`
```

Comment:
please remove the method in ErrorCatcher now which is still there

Reply:
Done in https://github.com/symfony/symfony/pull/32263